### PR TITLE
point to the correct acknowledgements file

### PR DESCRIPTION
**Pull request by joewalsh**

### CODE REVIEW

**Comment by joewalsh** [comment]
https://phabricator.wikimedia.org/T145763

File: Wikipedia/Code/AboutViewController.m [hunk 1]
```diff
@@ -259,7 +259,7 @@ - (void)webView:(WKWebView *)webView decidePolicyForNavigationAction:(WKNavigati
     NSURL *requestURL = navigationAction.request.URL;
 
     if ([[self class] isLicenseURL:requestURL]) {
-        VTAcknowledgementsViewController *vc = [VTAcknowledgementsViewController acknowledgementsViewController];
```
Comment (montehurd) [inline Wikipedia/Code/AboutViewController.m]: Hmm why did that stop working?

Comment (joewalsh) [inline Wikipedia/Code/AboutViewController.m]: Stopped working after CocoaPods config changed

**Comment by montehurd** [comment]
I see. Do you know what config? Wondering if we need to check for other places this may have broken things...

**Pull request merged**